lua/wikis/commons/TeamTemplate.lua

```diff
@@ -87,15 +89,15 @@ does not exist.
 ---@return teamTemplateData?
 function TeamTemplate.getRawOrNil(team, date)
 	team = team:gsub('_', ' '):lower()
-	return mw.ext.TeamTemplate.raw(team, date)
+	return Data[team] or mw.ext.TeamTemplate.raw(team, date)
```

Comment:
```suggestion
	return Data.specialTemplates[team] or mw.ext.TeamTemplate.raw(team, date)
```
The idea is to have it obvious on why we do a lookup of some Data module. When i first saw there being `Data[team]`, i thought it's used as some form of cache.